gradle-plugins/gradle.properties

```diff
@@ -7,6 +7,7 @@ kotlin.code.style=official
 #
 # __LATEST_COMPOSE_RELEASE_VERSION__
 compose.version=0.4.0-build182
+compose.with.web=false
```

Comment:
Why this change is required?

Reply:
It's not required per se. 
Our "with-web" plugin is expected to have additional properties & extensions, as we discussed previously (e.g. we discussed, that it should not be possible to add `compose.web.web` dependency with non-web plugin version).
We can implement the restrictions by literally checking a version number, but it seems more error prone and less intentional to me.

Reply:
Ack.